src/components/Promise/Home/AboutCard.svelte

```diff
@@ -59,21 +66,29 @@
 	</div>
 	<div class="flex flex-col gap-2 border-t border-ui-04 pt-3">
 		<p class="label-01 text-text-03">
-			รัฐบาลชุดที่ผ่านมาในสมัยการเลือกตั้ง {cabinet.startedAt.getFullYear() + 543}
+			รัฐบาลชุดที่ผ่านมาในสมัยการเลือกตั้ง {previousCabinet.startedAt.getFullYear() + 543}
 		</p>
 		<ul class="list-disc">
 			<li class="label-01 ml-5">
-				นายกรัฐมนตรี{primeMinister.firstname}
-				{primeMinister.lastname} และคณะรัฐมนตรีจาก
-				{#each cabinet.cabinetMemberCountsByParty as party}
-					{#if typeof party.party != 'string'}
-						{party.party.name} ({party.count} ตำแหน่ง),&nbsp;
+				นายกรัฐมนตรี{previousCabinet.primeMinister.firstname}
+				{previousCabinet.primeMinister.lastname} และคณะรัฐมนตรีจาก
+				{#each Array.from({ length: previousCabinetMemberCountsByParty.length }, (_, i) => i) as i}
+					{@const party = previousCabinetMemberCountsByParty[i]}
```

Comment:
I think you can turn these `each` and `const` into this which is more straightforward:
```svelte
{#each previousCabinetMemberCountsByParty as party, i}
```
Or I'm missing something?